setup.py

```diff
@@ -51,8 +51,8 @@
     },
 
     install_requires=[
-        'pylint==1.4.5',
-        'pylint-django==0.6.1',
+        'pylint>=1.5.4,<2.0.0',
```

Comment:
I'm not sure how pinning makes it fragile?  Part of the original goal of edx-lint was to ensure that all of our repos were linted the same.  It does that mostly by using a common pylintrc, but a pinned pylint version can also help.  So far, there doesn't seem to be a problem that is solved by unpinning pylint, so can we leave it pinned?